.github/workflows/build-image.yml

```diff
@@ -2,7 +2,8 @@
 name: Build and Publish ClamAV Image
 on:
     workflow_dispatch:
-    workflow_call:
+    schedule:
+      - cron: '0 4 * * 0'
```

Comment:
What is the issue number related to this PR ?
Is the intent to build this image once a week ?

Reply:
I don't know what the issue number is, but I do know that it is intended to have weekly image builds.

Reply:
@sambodeme https://github.com/GSA-TTS/FAC/issues/1064

Not sure what happened here: https://github.com/GSA-TTS/clamav-rest/actions/runs/5167122077 but testing this way for the schedule.

Reply:
@asteel-gsa  looks like the job run into a permission issue read vs write (which I believe is to be addressed outside of this PR). 
Regarding the weekly goal, this cron does achieve it (every Sunday 4am).